### PR TITLE
Variable rename, drop 'm_' prefix on mostly swing classes

### PR DESCRIPTION
**Pull request by DanVanAtta**
Since we do not serialize UI classes to disk or across network, it is safe to update variable names. On this branch a number were updated to have internal variables not have a 'm_' prefix (and there are one or two other quick small updates)

### CODE REVIEW

File: src/main/java/games/strategy/engine/framework/startup/ui/editors/EmailSenderEditor.java [hunk 1]
```diff
@@ -28,19 +28,19 @@
...
-  private final JLabel m_portLabel = new JLabel("Port:");
-  private final JButton m_testEmail = new JButton("Test Email");
-  private final JCheckBox m_alsoPostAfterCombatMove = new JCheckBox("Also Post After Combat Move");
+  private final GenericEmailSender genericEmailSender;
+  private final JTextField subject = new JTextField();
+  private final JTextField toAddress = new JTextField();
+  private final JTextField host = new JTextField();
+  private final JTextField port = new JTextField();
+  private final JTextField login = new JTextField();
+  private final JCheckBox useTLS = new JCheckBox("Use TLS encryption");
```
Comment (ssoloff) [inline src/main/java/games/strategy/engine/framework/startup/ui/editors/EmailSenderEditor.java]: If you feel like it, you could rename to `useTls` to also get rid of the abbreviation violation.

File: src/main/java/games/strategy/engine/framework/startup/ui/editors/SelectAndViewEditor.java [hunk 2]
```diff
@@ -29,14 +29,13 @@
...
-  JPanel m_view = new JPanel();
-  JButton m_helpButton = new JButton("Help?");
-  private final PropertyChangeListener m_properChangeListener;
-  private EditorPanel m_editor;
-  private final JLabel m_selectorLabel;
-  private final JEditorPane m_helpPanel;
-  private final String m_defaultHelp;
+  private JComboBox<IBean> selector = new JComboBox<>();
+  private JPanel view = new JPanel();
+  private final PropertyChangeListener properChangeListener;
```
Comment (ssoloff) [inline src/main/java/games/strategy/engine/framework/startup/ui/editors/SelectAndViewEditor.java]: It was already misspelled before this change, but `proper` --> `property`.

**Comment by ron-murhammer** [comment]
@DanVanAtta I'm fine with doing this refactoring but I'm pretty sure its going to conflict with #1853. So I'd like to have that merged first then rebase or redo this change since it just variable renamings.

**Comment by DanVanAtta** [comment]
Re-squashed and dropped conflicting classes from change set, latest has been pushed.

**Pull request merged**